pythonsd/templates/pythonsd/base.html

```diff
@@ -22,99 +21,93 @@
   </head>
 
   <body>
-    <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
-      {% url 'index' as index_url %}
-      <a class="navbar-brand" href="{% if request.path != index_url %}{{ index_url }}{% else %}#{% endif %}">
-        <img src="{% static 'img/sandiegopython-logo.svg' %}" alt="San Diego Python Home">
-      </a>
-      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbars-expand" aria-controls="navbars-expand" aria-expanded="false" aria-label="Toggle navigation">
-        <span class="navbar-toggler-icon"></span>
-      </button>
-
-      <div class="collapse navbar-collapse" id="navbars-expand">
-        <ul class="navbar-nav mr-auto">
-          <li class="nav-item">
-            <a class="nav-link" href="{% url 'code-of-conduct' %}">Code of Conduct</a>
-          </li>
-          <!-- <li class="nav-item">
-            <a class="nav-link" href="#">Support Us</a>
-          </li> -->
-          <li class="nav-item">
-            <a class="nav-link" href="{% url 'organizers' %}">Organizers</a>
-          </li>
-        </ul>
-      </div>
-    </nav>
+    <header class="bg-slate-700">
+      <nav class="container flex flex-wrap justify-between items-center py-6">
+        {% url 'index' as index_url %}
+        <a href="{% if request.path != index_url %}{{ index_url }}{% else %}#{% endif %}">
+          <img class="max-h-12" src="{% static 'img/sandiegopython-logo.svg' %}" alt="San Diego Python Home">
+        </a>
+
+        <!-- Navigation menu toggle -->
+        <div id="navigation-toggle" class="h-6 w-6 cursor-pointer md:hidden block text-slate-100">
+          <svg xmlns="http://www.w3.org/2000/svg" id="menu-button" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+            <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 6h16M4 12h16M4 18h16"></path>
+          </svg>
+        </div>
+
+        <div id="navigation-menu" class="hidden w-full mt-4 md:w-auto md:flex md:items-center md:mt-0">
+          <ul class="md:flex md:justify-between font-extrabold mb-0">
+            <li class="py-2 md:mr-5">
+              {% url 'code-of-conduct' as conduct_url %}
+              <a class="text-slate-100 hover:text-slate-300" href="{% if request.path != conduct_url %}{{ conduct_url }}{% else %}#{% endif %}">Code of Conduct</a>
+            </li>
+            <li class="py-2 md:mr-5">
+              {% url 'organizers' as organizers_url %}
+              <a class="text-slate-100 hover:text-slate-300" href="{% if request.path != organizers_url %}{{ organizers_url }}{% else %}#{% endif %}">Organizers</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
 
     <main role="main">
       {% block main %}{% endblock main %}
     </main>
 
     {# FOOTER #}
-    <footer class="container">
+    <footer class="container mx-auto">
```

Comment:
(Personal preference) I liked the old footer better on larger displays (more compact) but the new format looks better on mobile. (stacked lines and centered icons).

Reply:
This is fair. I think I like the old footer as well on larger displays although I'm not particularly worried about how compact it is. I think I like the look of the new icon spacing a bit better in the new footer but I don't love the spacing of text in the new footer.

I wonder if we could switch it to 2 text sections instead of 3. In reality, "edit on github" and the version could be condensed into one as they are related. See screenshots. Thoughts?

![Screenshot from 2024-06-22 07-46-33](https://github.com/sandiegopython/pythonsd-django/assets/185043/74923bcf-9adf-4603-adc4-fb77fb659665)
![Screenshot from 2024-06-22 07-46-53](https://github.com/sandiegopython/pythonsd-django/assets/185043/e5ef3769-b134-4691-8952-0867176e80c3)

Reply:
I merged this PR without changing this but I'd be happy to do a separate PR with this if you think this is an improvement.